index.html

```diff
@@ -9,27 +9,24 @@
 
   
 <body>
-  <div class="hero">
-    <header class="header bg-white">
-      <nav class="navbar flex">
-        <div class="navbar__logo">
-          <a href="#" class="navbar__link link fz-18 fw-700 lh-20">Mark.O</a>
-        </div>
-        <div class="navbar__menu">
-          <ul class="navbar__menu__ul ul flex">
-            <li class="navbar__menu__li">
-              <a href="#" class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600">Portfolio</a>
+  <nav class="navbar">
+    <a href="#" class="nav-branding">Mark.O</a>
+        <div class="nav-menu">
+            <li class="nav-item">
+              <a href="#works" class="nav-link">Portfolio</a>
             </li>
-            <li class="navbar__menu__li">
-              <a href="#" class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600">About</a>
+            <li class="nav-item">
+              <a href="#about" class="nav-link">About</a>
             </li>
-            <li class="navbar__menu__li">
-              <a href="#" class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600">Contact</a>
+            <li class="nav-item">
+              <a href="#contact" class="nav-link">Contact</a>
             </li>
           </ul>
         </div>
-        <div class="navbar__hamb">
-          <img src="logos/Union.jpeg" alt="menu icon" />
+        <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
         </div>
```

Comment:
Please stick to the Figma design and implement the `mobile menu` full height of the screen (Hint: you can use `height: 100vh`). Also, the texts should be aligned to the left with white color. It doesn't have to scrollable  it should be either fixed or sticky the only way the user can see the other part of the page is after choosing one opton from mobile menu or after closing the mobile menu.

| Your Design | Figma Design |
|-------------|---------------|
| <img width="349" alt="image" src="https://user-images.githubusercontent.com/34578631/174770133-1578967f-5f7c-4640-9792-f76151229afd.png"> | <img width="371" alt="image" src="https://user-images.githubusercontent.com/34578631/174770009-6bb070a5-7eb3-4353-af4c-05ca5ee31faf.png">|